test/Bin/CMakeLists.txt

```diff
@@ -19,3 +19,6 @@ add_subdirectory(UnitTest)
 # A target to build & run all tests in headless parallel mode.
 add_custom_target(Run_AllTests_Headless_Parallel
         DEPENDS Run_UnitTest Run_GameTest_Headless_Parallel Run_RetraceTest_Headless_Parallel)
+# Used for flatpak builds, would be nice to have retrace and game tests using Run_GameTest_Headless and Run_RetraceTest_Headless  ...
+add_custom_target(test
```

Comment:
Does flatpak just unconditionally run the `test` target? How does this work? With `run-tests: true`?

Reply:
Ah, OK, it's `ninja test`, I see it in the docs.

Reply:
This shouldn't work with `-DOE_BUILD_TESTS=OFF`.